### PR TITLE
v.1.4.0_SSR-2185-MobileDateTimePicker_component

### PR DESCRIPTION
**Pull request by voduchau**

### CODE REVIEW

File: src/mobile/datetime-picker/index.ts [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+    this._errorFormat = "";
+  }
+
+  private _getDateTimeValue(value: string | undefined) {
+    if (value === "" || value === undefined) return { date: "", time: "" };
+
+    const dateTime = value.split("T");
+    const date = dateTime[0];
+    const time = dateTime[1];
+    if (value.indexOf("T") === value.length - 1 || dateTime.length > 2)
```
Comment (nmanhit) [inline src/mobile/datetime-picker/index.ts:197]: what happen when value.indexOf("T") === -1 ?

File: src/mobile/datetime-picker/index.ts [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+    }
+    this._updateDateTimeValue(newValue, "date");
+  }
+
+  private _handleTimeChange(event: CustomEvent) {
+    event.preventDefault();
+    event.stopPropagation();
+    this._changeTimeByUI = true;
+    let newValue = this._timeValue;
+    if (event.detail.error) {
```
Comment (nmanhit) [inline src/mobile/datetime-picker/index.ts:299]:
```
if (event.detail.error) {
  this._errorFormat = event.detail.error;
  this.error = "";
} else {
  this._errorFormat = "";
}    
newValue = event.detail.value;
```

Comment (nmanhit) [inline src/mobile/datetime-picker/index.ts:299]:
```
if (event.detail.error) {
  this._errorFormat = event.detail.error;
  this.error = "";
} else {
  this._errorFormat = "";
}    
newValue = event.detail.value;
```

File: src/mobile/datetime-picker/index.ts (hunk 1 above)
Comment (voduchau) [reply src/mobile/datetime-picker/index.ts:197]: If the value don't have "T" the time part will be "00:00"

**Pull request merged**